docs/test/mftf-tests.md

```diff
@@ -0,0 +1,24 @@
+
+# MFTF Test Reference
+
+The Magento Functional Testing Framework runs tests on every Module within Magento.
+
+These files are stored within each Module folder in the Magento repository.
```

Comment:
The Test files will be moved out of Modules itself, but that's something we can update later when it's live.